src/mame/subsino/subsino.cpp

```diff
@@ -3358,6 +3358,53 @@ ROM_START( newhunterb )
 	ROM_LOAD( "ht-4.bin",     0x00345, 0x00117, CRC(f4f78925) SHA1(738281c05c0f51c0edcf65bacc18ebc87c2408c1) )
 ROM_END
 
+/*  Several custom modifications on this PCB:
+     -3V battery directly soldered to the main HM62256ALP-10 RAM.
+     -One 8-dipswitches bank soldered to a 74LS245.
+     -One 5-dipswitches bank soldered to another 74LS245.
+     -A single switch button soldered to another 74LS245.
+   And the usual hardware configuration:
+     -HD647180X0CP6.
+     -12 MHz xtal.
+     -1 x 8-dipswitches bank, plus the other two banks described before.
```

Comment:
Can you please write “DIP switches” in plain English?  “One bank of 8 DIP switches…” etc.

Reply:
Fixed, thanks!